### ISSUE TITLE
SHA1 is null

### ISSUE THREAD

**Issue by gameunite** [Backblaze/B2_Command_Line_Tool#368]
I tried uploading a 2gb file and check the contentSha1 and it says null. Is there an option in b2 sync. I searched about it and someone just added sending sha1 at the end of request but it seems it's not working now. or should I have to send an option or somethings.

Thanks

**Comment by svonohr** [comment]
When uploading large files they are uploaded as smaller parts of 100MB and only hashes of those individual parts are calculated and used for verifying each part. The B2 server does not calculate the sha1 of the full file in those cases. You can still calculate the hash yourself and specify it using the `--sha1 `option of the `upload-file` but whatever you specifiy here is not checked by the B2 server.

**Comment by gameunite** [comment]
Thanks for that info. Actually, I read that sha1 for large files will be stored as large_file_sha1. I tried to play around with the b2 options and it seems like as you've said that I have to input the sha1 manually which is kinda bummer. So I tried rclone copy and found out that they automatically do this computations for you. I notied that it's kinda slow compare to b2 in terms of uploading speed but that sha1 is really important for me so I just have to use rclone for now.

Thanks for your time.
